current_fall2025/scripts/gold_build_bpl_vector.py

```
@@ -0,0 +1,121 @@
+        processed_docs, inserted_chunks = 0, 0
+
+        for document_id, text, metadata, date_start, date_end in tqdm(rows, desc="🧠 Embedding documents"):
+            if not text:
+                continue
+
+            chunks = chunk_text(text)
+            if not chunks:
+                continue
+
+            embeddings = model.encode(chunks, batch_size=64, show_progress_bar=False)
+            for idx, (chunk, emb) in enumerate(zip(chunks, embeddings)):
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)